composer.json

```diff
@@ -5,10 +5,10 @@
     "description": "A Rich Editor plugin for Sylius.",
     "license": "MIT",
     "require": {
-        "php": "~7.4|~8.0",
+        "php": ">=8.1",
```

Comment:
This is a BC, right?
Keeping old versions is mandatory for Sylius 1.8+.

Can you adapt the change please ?

Thank you!!

J

Reply:
Hi @jacquesbh
I just made the necessary changes 😉